docs/src/user_docs/stdlib/crypto/hashes.md

```diff
@@ -9,6 +9,20 @@ Module `std::crypto::hashes::blake3` contains procedures for computing hashes us
 | hash_1to1   | Computes BLAKE3 1-to-1 hash.<br/><br/>Input: 32-bytes stored in the first 8 elements of the stack (32 bits per element).<br /> <br/>Output: A 32-byte digest stored in the first 8 elements of stack (32 bits per element). |
 | hash_2to1   | Computes BLAKE3 2-to-1 hash.<br/><br/>Input: 64-bytes stored in the first 16 elements of the stack (32 bits per element).<br /> <br/>Output: A 32-byte digest stored in the first 8 elements of stack (32 bits per element) |
 
+## RPO
```

Comment:
Let's change this to `RPO256` and move this section after `SHA256`.

Also, Let's fix a typo on line 2 above: `std::crypto` -> `std::crypto::hashes`.